article-analysis-project/article-processing.js

```diff
@@ -0,0 +1,44 @@
+//Modules
+const unfluff = require('unfluff');
+const cheerio = require('cheerio');
+const request = require('request');
+//Variables and Consts
+const url = 'https://abcnews.go.com/Politics/wireStory/canceling-ivanka-trump-spurs-backlash-kansas-college-71174430?cid=clicksource_4380645_2_heads_hero_live_headlines_hed'
+
+//Main
+
+request(url, (error, response, html) => {
+    if (!error && response.statusCode == 200){
+        console.log(html);
+        data = unfluff(html);
+        console.log(data.text);
+        console.log(data.author)
+        author = data.author
+        console.log(occurrences(data.text, " he ", false))
+    }
+
+})
+
+function occurrences(string, subString, allowOverlapping) {
```

Comment:
It's good practice to write documentation for functions.  In JavaScript there is a [standard for the DocBlock](https://devdocs.magento.com/guides/v2.4/coding-standards/docblock-standard-javascript.html) and depending on your IDE there is probably a plugin that will make it really easy to stub out documentation for a given function based on the parameters that already exist.